client/coral-admin/src/actions/community.js

```diff
@@ -41,15 +41,18 @@ export const newPage = () => ({
   type: COMMENTERS_NEW_PAGE
 });
 
-export const setRole = (id, role) => dispatch => {
-  return coralApi(`/users/${id}/role`, {method: 'POST', body: {role}})
+export const setRole = (id, role) => (dispatch, getState) => {
+
+  const _csrf = getState().auth.get('_csrf');
+  return coralApi(`/users/${id}/role`, {method: 'POST', body: {role}, _csrf: _csrf})
   .then(() => {
```

Comment:
This is also not part of the changes that were made. We can refactor in other PR.